.github/policies/issue.remove-needs-triage.yml

```diff
@@ -14,5 +15,31 @@ configuration:
       then:
       - removeLabel:
           label: 'Needs: Triage (Functions)'
+      triggerOnOwnActions: true
+
+    - description: Remove needs attention label from closed issues
+      if:
+      - payloadType: Issues
+      - isAction:
+          action: Closed
+      - hasLabel:
+          label: 'Needs: Attention :wave:'
+      then:
+      - removeLabel:
+          label: 'Needs: Attention :wave:'
+      triggerOnOwnActions: true
+
+    - description: Remove needs author feedback label from closed issues
+      if:
+      - payloadType: Issues
+      - isAction:
+          action: Closed
+      - hasLabel:
+          label: 'Needs: Author Feedback'
+      then:
+      - removeLabel:
+          label: 'Needs: Author Feedback'
+      triggerOnOwnActions: true
+
```

Comment:
Additional cleanup. The triggerOnOwnActions is meant as a catch for closures coming from these automations.